src/styles/theme.js

```diff
@@ -114,8 +114,9 @@ export default ({ color, backgroundColor, borderColor, cardColor }: Props) => ({
   link: {
     marginTop: 10,
     fontSize: 15,
+    padding: 5,
```

Comment:
To make this more universal, maybe increase it to `10` and remove `marginTop`?

Reply:
That could be done but I used the same design for the buttons. I'll update and ping you.

Reply:
Cool, that would be easy. I'll update and ping you.